docs/overview/02-getting-started/06-relayer.md

```diff
@@ -0,0 +1,205 @@
+    "vkRegistered": false,
+    "proofData": {
+        "proof": proofhex.split("\n")[0],
+        "publicSignals": pubhex.split("\n").slice(0,-1),
+        "vk": vkhex.split("\n")[0]
+    }
+}
+
+const requestResponse = await axios.post(`${API_URL}/submit-proof/${API_KEY}`, params)
+console.log(requestResponse.data)
+```
+</TabItem>
+</Tabs>
+
+After sending the verification request to the relayer, we can fetch the status of our request using the ``jobId`` returned in the response of the previous API call. To get the status, we will be making a ``GET`` API call to ``job-status`` endpoint. We want to wait till our proof is finalized on zkVerify, thus we will run a loop waiting for 5 seconds between multiple API calls.
+
+```js
+if(requestResponse.data.optimisticVerify != "success"){
+    console.error("Proof verification, check proof artifacts");
+    return;
+}
+
+while(true){
+    const jobStatusResponse = await axios.get(`${API_URL}/job-status/${API_KEY}/${requestResponse.data.jobId}`);
+    if(jobStatusResponse.data.status === "Finalized"){
+        console.log("Job finalized successfully");
+        console.log(jobStatusResponse.data);
+        break;
+    }else{
+        console.log("Job status: ", jobStatusResponse.data.status);
+        console.log("Waiting for job to finalize...");
+        await new Promise(resolve => setTimeout(resolve, 5000)); // Wait for 5 seconds before checking again
+    }
+}
+```
+
+Next run this script by running ``node index.js`` command. You should get a response similar to the following :- 
+```json
+{
+  jobId: '23382e04-3d57-11f0-af7b-32a805cdbfd3',
+  optimisticVerify: 'success'
+}
+Job status:  Submitted
+Waiting for job to finalize...
+Job status:  IncludedInBlock
+Waiting for job to finalize...
+Job status:  IncludedInBlock
+Waiting for job to finalize...
+Job finalized successfully
+{
+  jobId: '23382e04-3d57-11f0-af7b-32a805cdbfd3',
+  status: 'Finalized',
+  statusId: 4,
+  proofType: 'groth16',
+  chainId: null,
+  createdAt: '2025-05-30T13:08:11.000Z',
+  updatedAt: '2025-05-30T13:08:27.000Z',
+  txHash: '0xc0d85e5d50fff2bb5d192ee108664878e228d7fc3c1faa2d23da891832873d51',
+  blockHash: '0xcd574432b1a961305bbeb2c6b6ef399e1ae5102593846756cbb472bfd53d7d43',
+  transactionDetails: {}
+}
+```
+
+## Job Status
+
+In this example, we demonstrated how to wait for "Finalized" status for our proof verification. There are multiple proof status you can wait for. You can check all the status available following :- 
+
+- Queued - Proof accepted and waiting for processing
+- Valid - Proof passed optimistic verification
+- Submitted - Proof submitted to blockchain/mempool
+- IncludedInBlock - Proof transaction included in a block
+- Finalized - Proof transaction finalized on-chain
+- AggregationPending - Proof ready for aggregation
+- Aggregated - Proof successfully aggregated and published
+- AggregationPublished - Proof aggregation successfully published to zkVerify contract on destination chain
```

Comment:
I would mark these as optional, or somehow make it clear that there will not be an aggregation if chainId is not provided in the submit proof request